scripts/get

```diff
@@ -164,7 +173,6 @@ fail_trap() {
 # testVersion tests the installed client to make sure it is working.
 testVersion() {
   set +e
-  echo "$PROJECT_NAME installed into $HELM_INSTALL_DIR/$PROJECT_NAME"
```

Comment:
I moved this message just after `runAsRoot cp "$HELM_TMP_BIN" "$HELM_INSTALL_DIR"` in `installFile()` function.